### PR TITLE
Add recent searches based on local storage to home page

### PR DESCRIPTION
**Pull request by jaischeema**
Shows the most recent searches on the home page:

Screen Shot 2021-05-10 at 9 07 53 pm

### CODE REVIEW

**Comment by jaischeema** [comment]
Implements https://github.com/cashapp/logquacious/issues/46

File: app.sass [hunk 1]
```diff
@@ -34,7 +35,6 @@ html, body
   width: 100vw
   margin: 0
   padding: 0
-  overflow: hidden
```
Comment (jaischeema) [inline app.sass:37]: This is sort of unrelated but had to do this as it stops the home page from scrolling. It was also recently raised in Cash's internal chat.

File: src/services/Logquacious.ts [hunk 2]
```diff
@@ -66,6 +69,9 @@ export class Logquacious {
     }
 
     this.config = Logquacious.cleanConfig(this.config)
+
+    // TODO: Create appropriate tracker based on the config option
+    this.tracker = new BrowserStorageTracker(window.localStorage, this.config.filters)
```
Comment (jaischeema) [inline src/services/Logquacious.ts:74]: I expect the config option would be used to control what sort of tracker class should be initialized, have ignored that setup for now.

File: src/services/Logquacious.ts [hunk 3]
```diff
@@ -145,6 +151,20 @@ export class Logquacious {
     return config
   }
 
+  isTrackingEnabled(): boolean {
+    // TODO: Use the config option for this
+    // return this.config.tracker != null
```
Comment (cpakman) [inline src/services/Logquacious.ts:156]: remove this?

**Review by cpakman — APPROVED** [review]

**Pull request merged**